### PR TITLE
feat(db): add 3 columns to advertisment table that indicate various d…

### PR DESCRIPTION
**Pull request by ShaharAnati**
…ates regardign the ad

### CODE REVIEW

File: scripts/DDL.sql [hunk 1]
```diff
@@ -68,6 +68,9 @@ CREATE TABLE jobs.advertisements(
 	contact text,
 	base_location_id int REFERENCES jobs.base_locations(id) ON DELETE CASCADE,
 	is_close boolean DEFAULT false,
+	advertisment_date bigint,
```
Comment (stavbi1) [inline scripts/DDL.sql:71]: why not date? you need a timestamp?

File: server/routes/api/ads.js [hunk 2]
```diff
@@ -109,11 +109,11 @@ router.post('/', async (req, res) => {
     try {
         await db.query('BEGIN');
```
Comment (stavbi1) [inline server/routes/api/ads.js:110]: I know it's not your code, but why?

Comment (guyshany) [reply server/routes/api/ads.js:110]: The purpose of this line it to create a transaction (you can't send 2 SQL statements to the pool)

File: server/routes/api/ads.js [hunk 3]
```diff
@@ -109,11 +109,11 @@ router.post('/', async (req, res) => {
...
         
-        const values = [roleId, unitId, branchId, departmentId, jobNickname, jobDescription, entryDate, yearsInSeniority, shouldHaveDamach, req.user, `${contactInformation.fullName} ${contactInformation.phoneNumber}`, baseLocationId];
+        const values = [roleId, unitId, branchId, departmentId, jobNickname, jobDescription, entryDate, yearsInSeniority, shouldHaveDamach, req.user, `${contactInformation.fullName} ${contactInformation.phoneNumber}`, baseLocationId, advertismentDate, lastReferenceDate];
```
Comment (guyshany) [inline server/routes/api/ads.js:112]: Break those lines.. (add the prettier extension if it doesn't exist)

File: scripts/DDL.sql (hunk 1 above)
Comment (ShaharAnati) [reply scripts/DDL.sql:71]: Decided on number to save on conversions, how would you save it as Date in db and initialize with "now"?

Comment (SteveVaknin) [reply scripts/DDL.sql:71]: timestamp is better here IMO

Comment (michael-hoffman-26) [reply scripts/DDL.sql]: change it to timestamp

**Pull request merged**